src/runtime/virtcontainers/device/config/config.go

```diff
@@ -263,9 +263,15 @@ type VFIODev struct {
 	// Guest PCI path of device
 	GuestPciPath vcTypes.PciPath
 
+	// APDevices are the Adjunct Processor devices in the mdev. Mutually exclusive with PCI.
+	APDevices []string
+
 	// Type of VFIO device
 	Type VFIODeviceType
 
+	// IsAP specifies if device is AP or PCI
```

Comment:
Is this comment correct? It seems a bit odd that there are two `bool`'s to denote a PCI device.

Reply:
related to AP/PCI/PCIe discussion above